config/jobs/kubernetes/sig-testing/verify.yaml

```diff
@@ -38,8 +38,12 @@ presubmits:
         securityContext:
           privileged: true
         resources:
+          limits:
+            cpu: 7
```

Comment:
Is this the peak or the average? Not sure we want to reserve the peak CPU

Reply:
I was going off of recommended amount from @spiffxp. I _think_ I don't have access to metrics on these jobs, but could be wrong

Reply:
(really need to find a way to export this data)
![Screen Shot 2020-08-02 at 10 15 27 AM](https://user-images.githubusercontent.com/49258/89128242-209dde80-d4a9-11ea-91a1-84ca463c22e8.png)

Reply:
@liggitt @spiffxp ok to remove hold here?

Reply:
I'm ok with it as long as you're around to watch and adjust as needed if the jobs starts failing

Reply:
@spiffxp I will be :+1: